packages/lesswrong/components/surveys/SurveyScheduleEditPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { useCurrentUser } from "../common/withUser";
+import { useLocation, useNavigate } from "@/lib/routeUtil";
+import { Link } from "@/lib/reactRouterWrapper";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    paddingTop: 30,
+  },
+  surveyAdmin: {
+    color: theme.palette.primary.main,
+  },
+});
+
+const SurveyScheduleEditor = ({classes}: {
+  classes: ClassesType<typeof styles>,
+}) => {
+  const {params: {id}} = useLocation();
+  const navigate = useNavigate();
+  const isNewForm = !id;
+
+  const onCreate = useCallback(() => {
+    navigate("/admin/surveys");
+  }, [navigate]);
+
+  const {SingleColumnSection, SectionTitle, WrappedSmartForm} = Components;
+  return (
+    <SingleColumnSection className={classes.root}>
+      <Link to="/admin/surveys" className={classes.surveyAdmin}>
+        &lt;- Back to survey admin
+      </Link>
+      <SectionTitle title={`${isNewForm ? "New" : "Edit"} survey schedule`} />
+      <WrappedSmartForm
+        collectionName="SurveySchedules"
+        queryFragmentName="SurveyScheduleEdit"
+        mutationFragmentName="SurveyScheduleEdit"
+        documentId={id}
+        successCallback={onCreate}
+      />
```

Comment:
![Screenshot 2024-06-14 at 11 01 12](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/97f4cfc7-5d17-4f99-ae59-f710d4659b0f)

Could you make it so there is some way to copy the id of a schedule from the UI so it's possible to change the survey a schedule corresponds to?